scripts/associate_segments.sh

```diff
@@ -1,114 +0,0 @@
-#!/bin/bash
-
-DATA_DIR="/data/valhalla"
-
-# some defaults, if needed.
-export REGION=${REGION:-"us-east-1"}
-export OSMLR_DIR=${OSMLR_DIR:-"${DATA_DIR}/osmlr"}
-export NUMBER_OF_THREADS=${NUMBER_OF_THREADS:-"4"}
-
```

Comment:
in production, the nodes will use role based access control to access whatever resources. It’s better to assume that if there’s some need to run this locally for whatever reason that the person can just do `AWS_PROFILE=whatever some_command.sh`